src/Forms/Main.cs

```diff
@@ -1088,9 +1088,14 @@ private void Main_Load(object sender, EventArgs e)
                 if (screen.Bounds.Height < Height)
                     Height = screen.Bounds.Height;
 
-                if (screen.Bounds.X + screen.Bounds.Width - 200 < Left)

```

Comment:
MSDN: [Control-Left](https://msdn.microsoft.com/en-us/library/system.windows.forms.control.left(v=vs.110).aspx)

**Control.Left:** _Gets or sets the distance, in pixels, between the left edge of the control and the left edge of its container's client area._